### PR TITLE
istiocon 2022 announcement

### PR DESCRIPTION
**Pull request by therealmitchconnors**

### CODE REVIEW

**Comment by linsun** [comment]
LGTM, except one minor issue, could we get the formatting fixed?

image

**Review by linsun — APPROVED** [review]
LGTM, thank you @therealmitchconnors

File: content/en/blog/2022/istiocon-2022/index.md [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+    link="./istioconlogo.jpg"
+    alt="IstioCon logo"
+    >}}
+
+IstioCon provides an opportunity to showcase the lessons learned from running Istio in production, hands-on experiences from the Istio community, and will feature maintainers from across the Istio ecosystem. This year’s IstioCon features sessions focused on sharing real world examples, case studies, and success stories that can inspire newcomers to use Istio in production. The content will range from introductory to advanced levels, split into four main topic tracks:
+
+* **Getting started & getting involved**
+* **Tools, features & functionality** Observability, traceability, and other things built on top of Istio.
+* **Infrastructure & networking** Specific interest in talks addressing performance, cost, and multi-cloud environments.
+* **Tech evolution & what's next** The evolution of Istio, new standards, new extensions, and how to address problems that are interesting to tackle.
```
Comment (craigbox) [inline content/en/blog/2022/istiocon-2022/index.md:21]:
```suggestion
* **Tools, features & functionality:** Observability, traceability, and other things built on top of Istio.
* **Infrastructure & networking:** How Istio works, with deep-dives into performance, cost, and multi-cloud environments.
* **Tech evolution & what's next:** The evolution of Istio, new standards, new extensions, and how to address problems that are interesting to tackle.
```

**Review by craigbox — APPROVED** [review]

**Pull request merged**